testing/src/main/java/org/oppia/testing/TestCoroutineDispatcherRobolectricImpl.kt

```diff
@@ -2,13 +2,22 @@ package org.oppia.testing
 
 import kotlinx.coroutines.CancellableContinuation
 import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Delay
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.InternalCoroutinesApi
+import kotlinx.coroutines.TimeoutCancellationException
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.test.DelayController
 import kotlinx.coroutines.test.UncompletedCoroutinesError
+import kotlinx.coroutines.withTimeout
+import java.lang.IllegalStateException
```

Comment:
I don't know it's happening because of my some AS issue but removing this import brings no change in the file. Although we are using `IllegalStateException` in two places.

Reply:
Checked and your point is valid.

Reply:
java.lang is probably implicit--nice catch. Removed.

Reply:
Resolving this comment on behalf of @anandwana001 after verifying.